# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1363415202 14400
#      Sat Mar 16 02:26:42 2013 -0400
# Node ID 7612d75a559b7bfee6d7035987e70d9d7b460348
# Parent  09f0cb9cac7de0f3dbf662c2ff1951adad9fec28
distribute parser header files omitted in changeset faaf60aa457c

* libinterp/Makefile.am (octinclude_HEADERS): Include $(PARSER_INC) in
the list.
* parse-tree/module.mk (PARSER_INC): Delete parse-tree/parse-private.h
from the list.

diff --git a/libinterp/Makefile.am b/libinterp/Makefile.am
--- a/libinterp/Makefile.am
+++ b/libinterp/Makefile.am
@@ -105,16 +105,17 @@ EXTRA_DIST = \
 octinclude_HEADERS = \
   interpfcn/graphics-props.cc \
   parse-tree/oct-gperf.h \
   builtins.h \
   builtin-defun-decls.h \
   octave.h \
   $(OCTAVE_VALUE_INC) \
   $(PARSE_TREE_INC) \
+  $(PARSER_INC) \
   $(OPERATORS_INC) \
   $(INTERP_CORE_INC) \
   $(INTERPFCN_INC)
 
 nodist_octinclude_HEADERS = \
   interp-core/mxarray.h \
   interpfcn/defaults.h \
   interpfcn/graphics.h \
diff --git a/libinterp/parse-tree/module.mk b/libinterp/parse-tree/module.mk
--- a/libinterp/parse-tree/module.mk
+++ b/libinterp/parse-tree/module.mk
@@ -1,16 +1,15 @@
 EXTRA_DIST += \
   parse-tree/module.mk \
   parse-tree/octave.gperf
 
 PARSER_INC = \
   parse-tree/lex.h \
-  parse-tree/parse.h \
-  parse-tree/parse-private.h
+  parse-tree/parse.h
 
 PARSER_SRC = \
   parse-tree/lex.ll \
   parse-tree/oct-parse.yy
 
 PARSE_TREE_INC = \
   parse-tree/pt-all.h \
   parse-tree/pt-arg-list.h \
